tqdm/autonotebook.py

```diff
@@ -6,16 +6,13 @@
 >>> for i in trange(10):
 ...     ...
 """
-import os
 import sys
 from warnings import warn
 
 try:
     get_ipython = sys.modules['IPython'].get_ipython
     if 'IPKernelApp' not in get_ipython().config:  # pragma: no cover
         raise ImportError("console")
-    if 'VSCODE_PID' in os.environ:  # pragma: no cover
-        raise ImportError("vscode")
```

Comment:
do you know [which vscode version](https://github.com/tqdm/tqdm/issues/1213#issuecomment-1069714881) exactly this PR applies to? Is there a safer fix like:

```suggestion
    if os.environ.get('some_vscode_version_variable) <= '1337':  # pragma: no cover
        raise ImportError("vscode")
```

Reply:
I am afraid I do not know what version it applies to. Nor do I have any experience in knowing how to get the VS code version from the env!

Reply:
Ok will include as-is, thanks!

We'll see if anyone complains/suggests a fix later :)